Tiltfile

```diff
@@ -139,7 +138,18 @@ if MODE == "path_with_auth":
         watch=True,
     )
 
-    # 1. Build the External Auth Server image from envoy/auth_server/Dockerfile
+    # 1. Load the Kubernetes YAML for the envoy-proxy service
+    k8s_yaml("./local/kubernetes/envoy-proxy.yaml")
+    k8s_resource(
+        "envoy-proxy",
+        labels=["envoy_auth"],
+        # By default the Envoy Proxy container will bind to 127.0.0.1:3001.
+        # If you wish Envoy Proxy to be acessible outside the host machine,
+        # you must specify `port_forwards=["0.0.0.0:3001:3001"]`.
```

Comment:
Any reason we don't want to just make this the default for local development?

Reply:
No real reason; I will do that.